<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QDialog" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gra</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="headerGroup">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="planningTitle">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>371</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>aakar</family>
      <pointsize>26</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/3.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="endGameBtn">
    <property name="geometry">
     <rect>
      <x>1450</x>
      <y>30</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Zakończ grę</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>681</width>
      <height>72</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="infoGroup" stretch="10,75,10,15">
     <item>
      <widget class="QLabel" name="topicLabel">
       <property name="text">
        <string>Temat:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="topicTextEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="codeShowLabel">
       <property name="text">
        <string>Kod dostępu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="codeLineEdit">
       <property name="maxLength">
        <number>32000</number>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="playerNameLabel">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>10</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>aakar</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Admin</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="footerGroup">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>800</y>
     <width>1600</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>651</width>
      <height>72</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="manageTalkGroup" rowstretch="0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
     </property>
     <item row="0" column="1" rowspan="2">
      <widget class="QTextEdit" name="setTopicTextEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="setTopicLabel">
       <property name="text">
        <string>Ustaw temat dyskusji:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="setTopicBtn">
       <property name="text">
        <string>Zatwierdź</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="startTalkBtn">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="stopTalkBtn">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="resultLabel">
   <property name="geometry">
    <rect>
     <x>675</x>
     <y>740</y>
     <width>250</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>aakar</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="chatWidget" native="true">
   <property name="geometry">
    <rect>
     <x>1360</x>
     <y>100</y>
     <width>241</width>
     <height>701</height>
    </rect>
   </property>
   <widget class="QTextEdit" name="messageTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>211</width>
      <height>411</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="sendMessageLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendMessageBtn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>600</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyślij wiadomość</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Czat do rozmowy</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
